app/src/main/java/ru/vssemikoz/newsfeed/adapters/NewsFeedAdapter.java

```diff
@@ -34,14 +43,21 @@ public void setNewsList(List<NewsItem> newsList) {
     @Override
     public NewsFeedAdapter.NewsViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
         View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.news_item, parent, false);
-        return new NewsViewHolder(view);
+        return new NewsViewHolder(view, mListener);
     }
 
     @Override
     public void onBindViewHolder(@NonNull NewsViewHolder holder, int position) {
         NewsItem newsApiResponseItem = newsList.get(position);
```

Comment:
Why are we working with `newsApiResponseItem` here? I don't see any need for this. Let's rename it to `newsItem`.